examples/quickstart/run.py

```diff
@@ -50,19 +49,19 @@ def main():
     ).run()
 
     # initialize and run inference pipeline
-    inference_pipeline(
+    inference_pipeline_instance = inference_pipeline(
         inference_data_loader=inference_data_loader(),
         prediction_service_loader=prediction_service_loader(),
         predictor=predictor(),
         training_data_loader=training_data_loader(),
         skew_comparison=skew_comparison(),
         drift_detector=drift_detector,
-    ).run()
+    )
+    inference_pipeline_instance.run()
 
     # fetch latest runs for each pipeline
-    repo = Repository()
-    train_run = repo.get_pipeline(pipeline_name="training_pipeline").runs[-1]
-    inf_run = repo.get_pipeline(pipeline_name="inference_pipeline").runs[-1]
+    train_run = training_pipeline_instance.get_runs()[-1]
```

Comment:
We can't get the run that way right? Same as above

Reply:
Yes here we can't get it since `instance = pipeline(...).run()` will not work. Fixed it.